src/advanced_agent/database/connection.py

```
@@ -0,0 +1,295 @@
+    
+    def optimize_database(self) -> bool:
+        """データベース最適化"""
+        try:
+            with self.get_session() as session:
+                # VACUUM実行
+                session.execute(text("VACUUM"))
+                # ANALYZE実行
+                session.execute(text("ANALYZE"))
+                logger.info("データベース最適化完了")
+                return True
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)